backend/src/database/seeds/03-Posts.ts

```diff
@@ -13,11 +13,13 @@ export async function seed(knex: Knex): Promise<void> {
 
   const [alison, bob] = await knex.select<UserModel[]>().from('users');
 
+  console.log(alison, bob);
```

Comment:
remove console.log. also PRed these changes in the image upload PR, shld not have merge conn since the changes are the same

Reply:
ok!